packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -84,7 +89,19 @@ export class Runner {
         }
 
         this.logger.logInfo(`Writing page scan run result to a storage.`);
-        await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
+        const fullPageScanResult = await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
+
+        const featureFlags = await this.getDefaultFeatureFlags();
+        this.logger.logInfo(`sendNotification feature flag ${featureFlags.sendNotification}`);
+        if (featureFlags.sendNotification && !this.scanNotifyUrlEmpty(fullPageScanResult.notification)) {
+            this.logger.logInfo(`Sending notification to ${fullPageScanResult.notification.scanNotifyUrl}`);
+            await this.notificationDispatcher.sendNotificationMessage(this.createOnDemandNotificationRequestMessage(fullPageScanResult));
+        }
+    }
+
+    private scanNotifyUrlEmpty(notification: ScanCompletedNotification): boolean {
+        // tslint:disable-next-line: strict-boolean-expressions whitespace
```

Comment:
why should we disable this rule?

Reply:
because when I save (CTRL+S), a space is added automatic, not sure which extension does it

Reply:
remove this line & run yarn format:fix